infrastructure/templates/test-runner/image-template.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "[parameters('imageTemplateName')]",
+  "type": "Microsoft.VirtualMachineImages/imageTemplates",
+  "apiVersion": "2020-02-14",
+  "location": "[parameters('svcLocation')]",
+  "dependsOn": [],
+  "parameters": {
+    "imageTemplateName": {
+      "type": "string"
+    },
+    "svcLocation": {
+      "type": "string"
+    },
+    "managedImageResGroupId": {
+      "type": "string"
+    },
+    "managedImageName": {
+      "type": "string"
+    }
+  },
```

Comment:
This template doesn't look right - we need to follow the same format as the main `template.json`:

```json5
{
  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
  "contentVersion": "1.0.0.0",
  "parameters": {
  	// your parameters
  },
  "variables": {
  	// your variables
  },
  "resources": [
    {
      "type": "Microsoft.VirtualMachineImages/imageTemplates",
      "name": "[parameters('imageTemplateName')]",
       // etc
    }
  ]
}
``

Reply:
This hasn't been resolved. Also, it'd be good if the `variables` section could be at the top of the template rather than the bottom so it's more obvious where to find them.